src/java/org/apache/cassandra/repair/consistent/SyncStatSummary.java

```diff
@@ -147,21 +152,36 @@ public String toString()
             }
             StringBuilder output = new StringBuilder();
 
-            output.append(String.format("%s.%s - %s ranges, %s sstables, %s bytes\n", keyspace, table, ranges, files, FBUtilities.prettyPrintMemory(bytes)));
+            output.append(String.format("%s.%s - %s ranges, %s sstables, %s bytes\n", keyspace, table, ranges.size(), files, FBUtilities.prettyPrintMemory(bytes)));
```

Comment:
I'm unclear what the process is for modifying output of nodetool subcommands, but this will potentially change the console output of `nodetool repair`. IMO external tooling shouldn't be depending on the exact strings that nodetool emits, so this is fine with me but are we ok with having no option to force it to the old output?

Reply:
@beobal good point in general but I think this change is justifiable. It seems to be like overkill to me to offer the old format too.

Reply:
I agree, just wanted to mention it in case we are in the minority on this.